apps/api/src/app/environments-v1/usecases/create-environment/create-environment.usecase.ts

```diff
@@ -67,6 +97,21 @@ export class CreateEnvironment {
       });
     }
 
+    await this.createNovuIntegrationsUsecase.execute(
+      CreateNovuIntegrationsCommand.create({
+        environmentId: environment._id,
+        organizationId: environment._organizationId,
+        userId: command.userId,
+      })
+    );
```

Comment:
Moved the novu integration creation here as a concern of the environment setup to centralize it.

Reply:
partially related to this PR, why are we creating Novu integrations for non-development ?  
wouldn't adding the ability to test within the Development environment be enough?